Gemfile

```diff
@@ -42,6 +42,8 @@ gem 'bcrypt', '~> 3.1.7'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 # gem 'jbuilder', '~> 1.2'
 
+gem "bootstrap-sass"
```

Comment:
Let's be explicit about version here given that it's our views that are dependent on it. Tests could easily miss regressions, because the DOM will not have changed. So we should have to be deliberate about bumping this.